src/models/stores/stores.ts

```diff
@@ -35,6 +35,7 @@ export interface IBaseStores {
   supports: SupportsModelType;
   clipboard: ClipboardModelType;
   selection: SelectionStoreModelType;
+  isPreviewing?: boolean;
```

Comment:
I would place this higher up next to `appMode` since they're conceptually related.

Reply:
Got it - Done!